templates/workspaces/base/terraform/aad/variables.tf

```diff
@@ -1,3 +1,5 @@
 variable "key_vault_id" {}
 variable "workspace_resource_name_suffix" {}
 variable "workspace_owner_object_id" {}
+variable "tre_id" {}
+variable "tre_resource_id" {}
```

Comment:
instead of passing the "raw" string variables, you can (should) pass the map you already have in the parent module:
```terraform
variable tre_workspace_tags {}
```
And then use it as var.tre_workspace_tags in this module.